docs/Configuration.md

```diff
@@ -85,19 +85,50 @@ Note that certain chain options affect the format and indexing of the chain data
 ## HTTP
 
 - `http-host`: HTTP host to listen on (default: 127.0.0.1).
-- `http-port`: HTTP port to listen on (default: 8332).
+- `http-port`: HTTP port to listen on (default: 8332 for mainnet).
 - `ssl-cert`: Path to SSL cert.
 - `ssl-key`: Path to SSL key.
 - `service-key`: Service key (used for accessing wallet system only).
 - `api-key`: API key (used for accessing all node APIs).
-- `wallet-auth`: Enable token auth for wallets (default: false).
-- `no-auth`: Disable auth for API server and wallets (default: false).
 - `cors`: Enable "Cross-Origin Resource Sharing" HTTP headers (default: false).
 
 Note: For security `cors` should not be used with `no-auth`.\
 If enabled you should also enable `wallet-auth` and set `api-key`.
 
-## Sample Config File
+## Wallet options
+
+These options must be saved in `wallet.conf`:
+
+### bcoin client:
+
+- `node-host`: Location of bcoin node HTTP server (default: localhost).
```

Comment:
This is the websocket server, not HTTP?

Reply:
Well it serves both HTTP and websocket, but better to say RPC server here.

Reply:
Nevermind, this is consistent with other docs.

Reply:
Just checking in... This line is ok or needs a change?